net.efaxgtk.efaxgtk.yaml

```diff
@@ -0,0 +1,70 @@
+app-id: net.efaxgtk.efaxgtk
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: efax-gtk
+rename-desktop-file: efax-gtk.desktop
+rename-icon: efax-gtk
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=session-bus
+  - --share=network
+  - --filesystem=host
+  # For file browse dialogs
+  - --filesystem=xdg-run/gvfsd
+  # For modems
+  - --device=all
+cleanup:
+  - "/include"
+  - "*.la"
+  - "*.a"
+  - "/lib/debug"
+  # Shouldn't need these...
+  - "/lib/pkgconfig"
+  - "/share/doc"
+modules:
+  # Shamelessly stole tiff and ghostscript modules from gimp (and converted to yaml)
+  - name: libtiff
```

Comment:
libtiff is in the runtime. isn't it enough?

Reply:
Hrm, I'll try it out.

Reply:
Ok, yup, that seemed to work, thanks!